pkg/client/ghcr/ghcr.go

```diff
@@ -32,6 +34,12 @@ func New(opts Options) *Client {
 		panic(err)
 	}
 	client := github.NewClient(ghRateLimiter).WithAuthToken(opts.Token)
+	if opts.Hostname != "" {
+		client, err = client.WithEnterpriseURLs(fmt.Sprintf("https://%s/", opts.Hostname), fmt.Sprintf("https://%s/api/uploads/", opts.Hostname))
+		if err != nil {
+			panic(fmt.Errorf("setting enterprise URLs: %w", err))
```

Comment:
nit: reword error prefix, it sounds like the URLs are being set rather than failed to be set

Reply:
🙈  - Yea Assumed that the fact it was raised as an error - it was an error - but you're right, the error message prefix does need to be a little more explicit of the fact!